### PR TITLE
chore: updating dependabot to support TS, JS, Nuget and docker packages

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by amanji** [comment]
@rajpalc7 is this ready for review?

**Review by WadeBarnes — APPROVED** [review]
LGTM, but I'll leave it to the regular maintainers to perform the final review and approval.

**Review by amanji — APPROVED** [review]

**Pull request merged**
